### PR TITLE
Change some terms to unambiguous ones

### PR DESCRIPTION
**Pull request by yelizariev**
See https://github.com/odoo/odoo/pull/60765, https://github.com/odoo/odoo/pull/60810#discussion_r516625746 and commits descriptions

### CODE REVIEW

**Comment by yelizariev** [comment]
@mart-e can you approve terms changing for stable? Those terms are often confused by translator anyway

File: addons/event_sale/models/event.py [hunk 1]
```diff
@@ -249,7 +249,7 @@ def summary(self):
         order = self.sale_order_id.sudo()
         order_line = self.sale_order_line_id.sudo()
         if not order or float_is_zero(order_line.price_total, precision_digits=order.currency_id.rounding):
-            payment_status = _('Free')
+            payment_status = _('Free admission')
```
Comment (Yenthe666) [inline addons/event_sale/models/event.py:252]: I thought we didn't allow changing terms in a stable?

File: addons/web_editor/static/src/js/wysiwyg/widgets/crop_dialog.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ var CropImageDialog = Dialog.extend({
         this.$media = $(this.media);
         var src = this.$media.attr('src').split('?')[0];
         this.aspectRatioList = [
-            [_t("Free"), '0/0', 0],
+            [_t("Flexible"), '0/0', 0],
```
Comment (Yenthe666) [inline addons/web_editor/static/src/js/wysiwyg/widgets/crop_dialog.js:30]: Same here, thought we didn't allow changing terms in stable

File: addons/event_sale/models/event.py (hunk 1 above)
Comment (yelizariev) [reply addons/event_sale/models/event.py:252]: Yeah, but it's specific case. Check commit description and discussions

File: addons/hr_recruitment/models/hr_recruitment.py [hunk 3]
```diff
@@ -358,7 +358,7 @@ def action_get_attachment_tree_view(self):
     def action_applications_email(self):
         return {
             'type': 'ir.actions.act_window',
-            'name': _('Applications'),
+            'name': _('Candidacies'),
```
Comment (nim-odoo) [inline addons/hr_recruitment/models/hr_recruitment.py:361]: Not sure 'Candidacies' applies in this context.
```suggestion
            'name': _('Job Applications'),
```

File: addons/event_sale/models/event.py (hunk 1 above)
Comment (nim-odoo) [inline addons/event_sale/models/event.py:252]:
```suggestion
            payment_status = _('Free Admission')
```

**Review by nim-odoo — COMMENTED** [review]
@mart-e Although it is not ideal to change the terms in stable, in this case that would allow us to workaround the annoying translation limitation reported elsewhere.

It seems the most stable solution we can have.

About the ir_translation.py diff, I saw you and ODO have commented elsewhere. I didn't check it.

**Comment by mart-e** [comment]
I see how it avoids ambiguity and allow better translations based on the context but is it necessary to change it in 13.0 ? Do we have blocking ones in this version or is it just an annoyance ?
Because, as usual, merging this would mean the new messages will be untranslate.

**Comment by yelizariev** [comment]
These terms are either have wrong translations (in some languages) or the buggy translation framework will override one translation by another. So, it does't really makes things worse

**Comment by yelizariev** [comment]
Well, the solution is not not clean and resets these three terms back to English (from a correct translation or incorrect one), but it's not just clarifies terms for translators, it also allows to fix situation when correct translation is overridden by incorrect one (not because of translator mistake, but because of framework)

**Comment by yelizariev** [comment]
And without these terms changing, I'm sure that we can apply the framework fix (first commit): users may get wrong translation in a place where it worked before (because we make kind of change of translation priority).

But the framework fix is blocking (almost): "This week" and some other terms are not translated at all

**Comment by yelizariev** [comment]
As an option, we can apply this update in current version and then translate these 3 terms manually in transfer for all languages by using values for previous terms versions.

What do think? I can potentially do it myself, if I have access

**Comment by yelizariev** [comment]
@mart-e this should be fine for stable:

```
old = _('Free')
new = _('Free Admission')
new_is_translated = new != 'Free Admission'
payment_status = new if new_is_translated else old

```

File: addons/event_sale/models/event.py (hunk 1 above)
Comment (yelizariev) [reply addons/event_sale/models/event.py]: solved

**Comment by yelizariev** [comment]
@mart-e existing terms are not deleted anymore :-)

**Comment by mart-e** [comment]
My opinion is that the benefit is not worth the change in 13.0 but also it's not something that has that much importance and I am willing to fight for, I won't argue about it again. The new approach is quite hacky and if you want to go this road, be sure to remove it in the next version to avoid having to maintain it.

**Comment by yelizariev** [comment]
@robodoo retry

**Comment by yelizariev** [comment]
@mart-e can you review first commit? https://github.com/odoo/odoo/pull/61272/commits/15ca00ce139634ab6d1113dc5f116dbd005e8411

**Comment by pedrobaeza** [comment]
@mart-e I receive complaints from my customers asking why certain terms are not translated, so if we can avoid it ASAP, better than leave it for v15.

**Comment by nim-odoo** [comment]
For the context of this bug, I would only keep the "clarify english term" commits without the hacky part. We know we break the translations, but that's for the best here.

We can leave the "This week" term untranslated, or at least it can be merged in master since this can have side effects.

**Comment by mart-e** [comment]
> @mart-e can you review first commit? 15ca00c

That looks fine :+1: It doesn't seem to impact the performances. With the unicity constraint on code translation, we accept only one translation so we have to live with the fact that some translations will be lost in case of different translation in different modules.

File: odoo/addons/base/models/ir_translation.py [hunk 4]
```diff
@@ -109,9 +109,11 @@ def finish(self):
...
                            ON CONFLICT (type, lang, md5(src)) WHERE type = 'code'
-                            DO UPDATE SET (name, lang, res_id, src, type, value, module, state, comments) = (EXCLUDED.name, EXCLUDED.lang, EXCLUDED.res_id, EXCLUDED.src, EXCLUDED.type, EXCLUDED.value, EXCLUDED.module, EXCLUDED.state, EXCLUDED.comments)
-                            WHERE EXCLUDED.value IS NOT NULL AND EXCLUDED.value != '';
-                       """ % (self._model_table, self._table))
+                            DO UPDATE SET (name, lang, res_id, src, type, value, module, state, comments) = (EXCLUDED.name, EXCLUDED.lang, EXCLUDED.res_id, EXCLUDED.src, EXCLUDED.type, EXCLUDED.value, EXCLUDED.module, EXCLUDED.state,
+                                                                                                             CASE WHEN %s.comments = 'openerp-web' THEN 'openerp-web' ELSE EXCLUDED.comments END
+                                                                                                            )
+                            WHERE EXCLUDED.value IS NOT NULL AND EXCLUDED.value != ''
+                       """ % (self._model_table, self._table, self._model_table))
```
Comment (yelizariev) [inline odoo/addons/base/models/ir_translation.py:116]: @odony in your [patch](https://github.com/odoo/odoo/pull/60810#discussion_r515311762) it was

                       """ % (self._model_table, self._table, self._table))

But having ``self._table`` at the end of the list makes another statement equal to 

      CASE WHEN EXCLUDED.comments = 'openerp-web' THEN 'openerp-web' ELSE EXCLUDED.comments END

Am I correct? Should it be ``self._model_table`` as I did in this PR?

**Comment by yelizariev** [comment]
I moved changes in base module ("the first commit") to a separate PR  https://github.com/odoo/odoo/pull/61641

Also, I made a PR for master with the same update: https://github.com/odoo/odoo/pull/61640

File: odoo/addons/base/models/ir_translation.py (hunk 4 above)
Comment (nim-odoo) [reply odoo/addons/base/models/ir_translation.py]: ping @odony

**Comment by yelizariev** [comment]
Also, opw-2371196

**Comment by yelizariev** [comment]
PR for master (Updates terms "Free" and "Applications"): https://github.com/odoo/odoo/pull/61668

**Pull request closed without merging**